header.js

```diff
@@ -0,0 +1,56 @@
+/*
+
+This is a prototype Javascript API for Sketch.
+
+The intention is to make something which is:
+
+- native Javascript
+- an easily understandable subset of the full internals of Sketch
+- fully supported by Bohemian between releases (ie we try not to change it, unlike our internal API which we can and do change whenever we need to)
+- still allows you to drop down to our internal API when absolutely necessary.
+
+Comments and suggestions for this API are welcome - send them to developer@sketchapp.com.
```

Comment:
developers@sketchapp.com (we should probably set up a forwarder from one address to the other, since I can't remember which one is the one we're using : )